shared/message/strategy/Ready.proto

```diff
@@ -20,10 +20,5 @@ syntax = "proto3";
 
 package message.strategy;
 
-import "Vector.proto";
-
 /// Ready state strategy for walking onto the field
-message Ready {
-    /// The target position to walk to relative to the field
-    vec3 rRFf = 1;
```

Comment:
Could we use a target position now that we have localisation?

Reply:
yes we can but not doing it in this PR, can come in a future one

Reply:
I didn't want to have things lying around that weren't being used, like this. It might not be the best way to organise localisation based ready. We probably want a different message and module, something like `WalkToFieldPosition`. And if we keep the old basic ready, we can easily switch it in if localisation stops working.